### DISCUSSION TITLE
Module graph

### DISCUSSION THREAD

**Discussion by underfisk** [vitest-dev/vitest#1372] [Q&A]
I have been playing around with the module graph and I was curious if the reported modules are actually duplicated or if it's just an analysis of the file imports because it looks like lodash is represented more than once and some other packages (mostly `@mui` related).

A simple example that I have created is a `<RootComponent>` that renders a `<ChildComponent>`. If the Child component imports Box from MUI and Root also imports the same or any other component from MUI, the graph will display twice (or even more sometimes) and it starts to be confusing

**Comment by sheremet-va** [comment]
I think this is actually a bug. Can you create an issue with reproduction?
